### PR TITLE
add argo-models

### PR DESCRIPTION
**Pull request by nicolas-geniteau**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by nicolas-geniteau** [comment]
> * The license item is expected in the about section.

There is no license in the argo-models repository. What should I do ?

**Comment by marcelotrevisani** [comment]
> > * The license item is expected in the about section.
> 
> There is no license in the argo-models repository. What should I do ?

You can add the license manually. Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

File: recipes/argo-models/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=2.7,!=3.0,!=3.1,!=3.2,!=3.3
```
Comment (marcelotrevisani) [inline recipes/argo-models/meta.yaml]: you can let it just
```suggestion
    - python >=3
```

Comment (marcelotrevisani) [reply recipes/argo-models/meta.yaml]: Because our CI will create packages just for Python 2.7, 3.6 and 3.8

**Comment by nicolas-geniteau** [comment]
> I see that there is no `license` specified on their github repository. You can open an issue there asking for which license they are using and also request to package the license along with their package

Thanks, I just opened an issue.

**Comment by nicolas-geniteau** [comment]
> Thanks, I just opened an issue.

They added the license in the package.

@conda-forge/help-python ready for review.

**Pull request merged**